### DISCUSSION TITLE
Включение отладки

### DISCUSSION THREAD

**Discussion by michael-sosin** [totumonline/totum-ru-issues-and-discussions#627] [Как это сделать?] [answered]
Доброго времени суток. Подскажите пожалуйста, как включить отладку из статьи https://ru.docs.totum.online/debugging#sx-enabling-debugging. Данная плашка не отображается 😿 

![image](https://github.com/user-attachments/assets/a7ea39e7-c361-480d-b8d3-821ec65ec915)

**Comment by erpic-ru** [comment, marked as answer]
Не отображается из-за размера экрана. Скройте боковую панель с деревом, либо измените масштаб страницы

**Reply by michael-sosin** [reply]
А ларчик просто открывался. Спасибо.
